larray/core/session.py

```diff
@@ -308,6 +448,14 @@ def names(self):
         Returns
         -------
         list of str
+        
+        Examples
+        --------
+        >>> arr1, arr2, arr3 = ndtest((2, 2)), ndtest(4), ndtest((3, 2))
+        >>> s = Session([('arr1', arr1), ('arr2', arr2), ('arr3', arr3)])  
+        >>> # print array's names in sorted order                    
```

Comment:
Shouldn't the mention of being sorted be in the description rather than in the example? I wonder if we shouldn't mention explictly that this is the same than keys() except for the sorted part. Maybe in a See also section?